ros/humble.v3.17/ros-humble-nav2-constrained-smoother/apkbuild_hook.sh

```diff
@@ -0,0 +1,4 @@
+apkbuild_hook() {
+  makedepends="${makedepends} blas-dev glog-dev lapack-dev libtbb-dev metis metis-dev suitesparse-dev xtl"
+  depends="${depends} blas-dev glog-dev lapack-dev libtbb-dev metis suitesparse-dev xtl"
```

Comment:
Is it an upstream problem? (missing deps in manifest?)

Reply:
These packages are needed to build with `celes-solver`. It might be better to add them to `depends` of `ceres-solver`.

Reply:
I found that some of these packages are actually unnecessary. I'll clean up the list before merging.

Reply:
So, I guess they should be `depends_dev` of celes-solver

Reply:
https://github.com/seqsense/aports-ros-experimental/pull/904